source/guides/maintenance/cpu-performance.rst

```diff
@@ -155,34 +143,49 @@ The list of all governors can be found in the Linux kernel documentation on
 
    The intel_pstate driver only supports *performance* and *powersave* governors.
 
+There are 2 ways to change the CPU frequency scaling governor:
+
+#. Disable |CL| enforcement of certain power and performance settings:
+
+   .. code-block:: bash
+
+      sudo systemctl mask clr-power.timer
+
+#. Change the governor value in :file:`/sys/devices`. In the example below,
+   the governor is set to *performance*:
+
+   .. code-block:: bash
+
+      echo performance | sudo tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor
+
 Thermal management
 ******************
 
-`thermald`_ is a Linux thermal management daemon used to prevent the
-overheating of platforms. When temperature thresholds are exceeded, thermald
-forces a C-state by inserting CPU sleep cycles and adjusts available cooling
-methods. This can be especially desirable for laptops.
+`thermald`_ is a Linux thermal management daemon used to prevent platforms 
+from overheating. :command:`thermald` forces a C-state by inserting CPU sleep
+cycles and adjusting any available cooling methods. This can be especially
+desirable for laptops.
 
-By default, thermald is disabled in |CL| and starts automatically if battery
-power is detected. thermald can be manually enabled using the systemd service
-by running the command:
+:command:`thermald` is disabled by default in |CL| and starts automatically
+if it detects battery power. Enable :command:`thermald` manually by using
+the systemd service by running the command:
 
-.. code:: bash
+.. code-block:: bash
 
    sudo systemctl enable --now thermald
 
-For more information, see the thermald man page:
+For more information, see the :command:`thermald` man page:
 
-.. code:: bash
+.. code-block:: bash
 
    man thermald
```

Comment:
Ln 181: add `sudo`. 
Requires root privileges.

Reply:
Listing man page should not require sudo.

Reply:
You're right. No need here.